diff -uNr 7.0-orig/aux-file-data.sh 7.0/aux-file-data.sh
--- 7.0-orig/aux-file-data.sh	2012-02-26 16:10:14.941373241 -0600
+++ 7.0/aux-file-data.sh	2012-02-26 16:11:13.965665932 -0600
@@ -7,6 +7,8 @@
 
 FILE=$1
 
+./make-aux-files.sh
+
 # Bootscript data
 bootscripts=$(ls lfs-bootscripts*.bz2)
 base=$(basename $bootscripts .tar.bz2)
Binary files 7.0-orig/lfs-bootscripts-20111017.tar.bz2 and 7.0/lfs-bootscripts-20111017.tar.bz2 differ
diff -uNr 7.0-orig/Makefile 7.0/Makefile
--- 7.0-orig/Makefile	2012-02-26 16:10:13.085364043 -0600
+++ 7.0/Makefile	2012-02-26 16:25:16.317842947 -0600
@@ -12,7 +12,7 @@
   Q = @
 endif
 
-lfs: maketar validxml profile-html
+lfs: validate profile-html
 	@echo "Generating chunked XHTML files..."
 	$(Q)xsltproc --nonet -stringparam chunk.quietly $(CHUNK_QUIET) \
 	  -stringparam rootid "$(ROOT_ID)" -stringparam base.dir $(BASEDIR)/ \
@@ -40,7 +40,7 @@
 
 	$(Q)$(MAKE) wget-list
 
-pdf: validxml
+pdf: validate
 	@echo "Generating profiled XML for PDF..."
 	$(Q)xsltproc --nonet --stringparam profile.condition pdf \
 	  --output $(RENDERTMP)/lfs-pdf.xml stylesheets/lfs-xsl/profile.xsl \
@@ -59,7 +59,7 @@
 	fi;
 	$(Q)fop $(RENDERTMP)/lfs-pdf.fo $(BASEDIR)/$(PDF_OUTPUT)
 
-nochunks: maketar validxml profile-html
+nochunks: validate profile-html
 	@echo "Generating non chunked XHTML file..."
 	$(Q)xsltproc --nonet -stringparam rootid "$(ROOT_ID)" \
 	  --output $(BASEDIR)/$(NOCHUNKS_OUTPUT) \
@@ -78,7 +78,7 @@
 	$(Q)rm -f $(RENDERTMP)/lfs-{full,html,pdf}.xml
 	$(Q)rm -f $(RENDERTMP)/lfs-pdf.fo
 
-validxml: tmpdir
+validate: tmpdir
 	@echo "Processing bootscripts..."
 	$(Q)bash process-scripts.sh
 	@echo "Validating the book..."
@@ -86,14 +86,9 @@
 	  -o $(RENDERTMP)/lfs-full.xml index.xml
 	$(Q)rm -f appendices/*.script
 	$(Q)./aux-file-data.sh $(RENDERTMP)/lfs-full.xml
+	@echo "Validation complete."
 
-maketar:
-	$(Q)if [ "x$(MAKETAR)" == "x" ]; then \
-	   echo "Making tarballs..."; \
-	   sh make-aux-files.sh; \
-	fi;
-
-profile-html: validxml
+profile-html: validate
 	@echo "Generating profiled XML for XHTML..."
 	$(Q)xsltproc --nonet --stringparam profile.condition html \
 	  --output $(RENDERTMP)/lfs-html.xml stylesheets/lfs-xsl/profile.xsl \
@@ -116,15 +111,12 @@
       $(BASEDIR)/md5sums
 
 
-dump-commands: validxml
+dump-commands: validate
 	@echo "Dumping book commands..."
 	$(Q)xsltproc --output $(DUMPDIR)/ \
 	   stylesheets/dump-commands.xsl $(RENDERTMP)/lfs-full.xml
 
-validate: maketar validxml
-	@echo "Validation complete."
-
 all: lfs nochunks pdf dump-commands md5sums
 
-.PHONY : all dump-commands lfs nochunks pdf profile-html tmpdir validate \
-	 validxml wget-list maketar md5sums
+.PHONY : all dump-commands lfs nochunks pdf profile-html tmpdir \
+	 validate wget-list md5sums
Binary files 7.0-orig/udev-config-20100128.tar.bz2 and 7.0/udev-config-20100128.tar.bz2 differ
